lib/SyTest/Homeserver/Synapse.pm

```diff
@@ -1269,6 +1271,10 @@ EOCONFIG
       $haproxy_map .= <<'EOCONFIG';
 
 ^/_matrix/client/(api/v1|r0|unstable)/sendToDevice/          client_reader
+^/_matrix/client/(api/v1|r0|unstable)/rooms/.*/tag           client_reader
```

Comment:
Isn't this `tags`, not `tag`?

Reply:
It's `/tag/:type` and `/tags$`, and this (rather cheekily) covers both

Reply:
Ah, right. 👍